src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -77,20 +85,100 @@ export const survivalClinicalDataNullValueSet = new Set([
     'na',
 ]);
 
-export function getEstimates(patientSurvivals: PatientSurvival[]): number[] {
-    let estimates: number[] = [];
+export function getSurvivalSummaries(
+    patientSurvivals: PatientSurvival[]
+): SurvivalSummary[] {
+    let summaries: SurvivalSummary[] = [];
     let previousEstimate: number = 1;
+    let previousMonths: number | undefined = undefined;
+    let isErrorCalculatable: boolean = false;
+    let previousError: number = 0;
+
+    // calculate number of events
+    let numberOfEvenrts: number = 0;
+    const nEventDict: Dictionary<number> = _.reduce(
```

Comment:
seems like you are producing "counts" so i would include the word count in naming

Reply:
changed to `eventCountsByMonths`